src/argilla/client/feedback/dataset.py

```diff
@@ -759,13 +759,12 @@ def format_as(self, format: Literal["datasets"]) -> "Dataset":
                     dataset[question.name].append(
                         [
                             {
-                                "user_id": r.user_id,
-                                "value": r.values[question.name].value,
-                                "status": r.status,
+                                "user_id": r.user_id or None,
+                                "value": r.values[question.name].value or None,
```

Comment:
this may fail when the value is a `0`. In those cases, the returning value will be `None, ' which is unexpected. Can you be more explicit in setting the default value for user_id, value, and status?

Reply:
Nice catch, didn't think about the `0 or None` case, but you're right!